docker-app/qfieldcloud/core/models.py

```diff
@@ -1218,6 +1239,58 @@ class Meta:
         default=False,
     )
 
+    def get_localized_datasets_project(self) -> Optional[Project]:
+        """
+        Returns the 'localized_datasets' Project instance for the same owner,
+        or None if no such project exists.
+        """
+        try:
+            project: Project = Project.objects.only("id").get(
```

Comment:
```suggestion
            project: Project = Project.objects.get(
```
Since now we return the whole project, I guess we don't need to select only the id?

Reply:
Also do we need the explicit typing for the `project` variable? I am surprised we don't have the correct time from `Project.objects.get()`.

Reply:
No need for this!